homeassistant/components/stiebel_eltron/__init__.py

```diff
@@ -0,0 +1,77 @@
+"""
+The component for STIEBEL ELTRON heat pumps with ISGWeb Modbus module.
+
+Example configuration:
+
+modbus:
+  type: tcp
+  host: YOUR_ISGWEB_IP
+  port: 502
+
+stiebel_eltron:
+  name: LWZ504e
+
+For more details about this component, please refer to the documentation
+https://home-assistant.io/components/stiebel_eltron/
```

Comment:
We don't add the url in the docstring anymore. Just keep the very first line so that the docstring is a single line.

Reply:
Done